spec.html

```diff
@@ -733,331 +658,100 @@ <h1>Example Source Text Module Record Graphs</h1>
 
-    <p>Now consider a module graph with a cycle:</p>
+    <p>Lastly, consider a module graph with a cycle:</p>
 
     <emu-figure id="figure-module-graph-cycle" caption="A cyclic module graph">
       <img alt="A module graph in which module A depends on module B and C, but module B also depends on module A" width="181" height="121" src="img/module-graph-cycle.svg">
     </emu-figure>
 
-    <p>Here we assume that the entry point is module _A_, so that the host proceeds by calling _A_.Link(), which performs InnerModuleLinking on _A_. This in turn calls InnerModuleLinking on _B_. Because of the cycle, this again triggers InnerModuleLinking on _A_, but at this point it is a no-op since _A_.[[Status]] is already `"linking"`. _B_.[[Status]] itself remains `"linking"` when control gets back to _A_ and InnerModuleLinking is triggered on _C_. After this returns with _C_.[[Status]] being `"linked"` , both _A_ and _B_ transition from `"linking"` to `"linked"` together; this is by design, since they form a strongly connected component.</p>
+    <p>Here we assume that the entry point is module _A_, so that the host proceeds by calling _A_.Link(), which performs InnerModuleLinking on _A_. This in turn calls InnerModuleLinking on _B_. Because of the cycle, this again triggers InnerModuleLinking on _A_, but at this point it is a no-op since _A_.[[Status]] is already ~linking~. _B_.[[Status]] itself remains ~linking~ when control gets back to _A_ and InnerModuleLinking is triggered on _C_. After this returns with _C_.[[Status]] being ~linked~ , both _A_ and _B_ transition from ~linking~ to ~linked~ together; this is by design, since they form a strongly connected component.</p>
 
     <p>An analogous story occurs for the evaluation phase of a cyclic module graph, in the success case.</p>
 
-    <p>Now consider a case where _A_ has an linking error; for example, it tries to import a binding from _C_ that does not exist. In that case, the above steps still occur, including the early return from the second call to InnerModuleLinking on _A_. However, once we unwind back to the original InnerModuleLinking on _A_, it fails during ModuleDeclarationEnvironmentSetup, namely right after _C_.ResolveExport(). The thrown *SyntaxError* exception propagates up to _A_.Link, which resets all modules that are currently on its _stack_ (these are always exactly the modules that are still `"linking"`). Hence both _A_ and _B_ become `"unlinked"`. Note that _C_ is left as `"linked"`.</p>
```

Comment:
Please restore all my examples!

Reply:
Of course! I didn't realize they were additions. I tried to transfer everything that was `<ins>`

This is a meta discussion, personally I think doing this by hand is too much (this is the first time I do it, might be an experience thing). It results in unspoken conventions that might not be recognized by new reviewers. It feels like -- we have diffing systems, They do this much better than humans. It would be great to be able to use them to our advantage and have a base `spec.html` file, along with a `diff` file, and have ecmarkup recognize +/- at the beginning of a line. It would _really_ help situations like this. Then we could have a plain spec.html file which has the spec as it would be written, and a spec-diff.html file that shows the changes from the branch point. But again, not something I have done very often so it would be interesting to hear what other people think.

Reply:
Hm, it doesnt look like this line is different beyond the changes to "linked" -> ~linked~ and similar?